benchmarks/distributed_tree_driver/distributed_tree_driver.cpp

```diff
@@ -285,22 +291,16 @@ int main_(std::vector<std::string> const &args, const MPI_Comm comm)
               << "#queries/MPI process    : " << n_queries << '\n'
               << "size of shift           : " << shift << '\n'
               << "dimension               : " << partition_dim << '\n'
+              << "shift-queries           : " << shift_queries << '\n'
               << '\n';
   }
 
-  Kokkos::View<ArborX::Point *, DeviceType> random_points("random_points", 0);
+  Kokkos::View<ArborX::Point *, DeviceType> random_values(
```

Comment:
Bad name

Reply:
I agree. However, it is aligned with `--values` cmd line argument. Personally, i would change both values and queries cmd line names to something like `--num_points` and `--num_queries` both here and bch_driver.